doc/source/templates/04_finetuning_llms_with_deepspeed/run_llama_ft.sh

```diff
@@ -75,9 +75,13 @@ do
     key=${arg%%=*}
     value=${arg#*=}
     if [[ "$key" == "--size" ]]; then
-        SIZE=${value}
-    elif [ "$arg" = "--as-test" ]; then
-        params+=("--as-test")
+        SIZE=${value};
+    elif [[ "$arg" == "--as-test" ]]; then
+        params+=("--as-test");
+    elif [[ "$arg" == "--lora" ]]; then
+        params+=("--lora");
+        params+=("--lr");
+        params+=("1e-4");
```

Comment:
add `--lora --lr 1e-4` in a single line? with a comment that the learning rate is scaled up for LoRA.